components/cta_bar/index.styl

```diff
@@ -1,14 +1,69 @@
 @import '../stylus_lib'
 
+.cta-bar.editorial-signup
+  padding 0
+  background none transparent
+  background linear-gradient(to bottom, rgba(0,0,0,0), rgba(0,0,0,.25))
+  display flex
+  align-items flex-end
+  min-height 35vh
```

Comment:
hmm what's this for? might be nicer to set a specific height instead from the Sketch file.